spec/factories.rb

```diff
@@ -1,5 +1,26 @@
-FactoryGirl.define do  factory :user do
-    
+FactoryGirl.define do  factory :job do
+    type nil
+appointment nil
+status 1
+  end
```

Comment:
`end` at 5, 2 is not aligned with `factory :job do` at 1, 23 or `FactoryGirl.define do  factory :job do` at 1, 0.